website/docs/docs/develop/dbt-cloud-tips.md

```diff
@@ -0,0 +1,115 @@
+- Use the `+` operator on the left of a model `dbt build +model_name` to run a model and all of its upstream dependencies. Use the `+` operator on the right of the model `dbt build model_name+` to run a model and everything downstream that depends on it.
+- Use `dir_name` to run all models in a package or directory.
+- Use the `@` operator on the left of a model in a non-state-aware CI setup to test it. This operator runs all of a selection’s parents and children, and also runs the parents of its children, which in a fresh CI schema will likely not exist yet.
+- Use the [`--exclude` flag](/docs/reference/node-selection/exclude) to remove a subset of models out of a selection.
+- Use [`state and deferral`](/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration#deferral-and-state-comparison) to create a slim CI setup.
+- Use the [`--full-refresh`](/docs/reference/commands/run#refresh-incremental-models) flag to rebuild an incremental model from scratch.
+- Use [`seeds`](/docs/building-a-dbt-project/seeds) to create manual lookup tables, like zip codes to states or marketing UTMs to campaigns. `dbt seed` will build these from CSVs into your warehouse and make them `ref` able in your models.
+- Use [`target.name`](/docs/building-a-dbt-project/building-models/using-custom-schemas#an-alternative-pattern-for-generating-schema-names) to pivot logic based on what environment you’re using. For example, to build into a single development schema while developing, but use multiple schemas in production.
+
+## New Cloud IDE features
+
+- **Version control**
+    - [NOT BUILT YET] Git diff view: Ability to see what has been changed in a file before you make a pull request
+    - [NOT BUILT YET] Merge conflict: Improve ways to deal with merge conflict
+- **IDE organization**
+    - **File state indicators:** The indicators **M, U, and •** appear to the right of your file or folder name. They indicate the action performed:
+        - Unsaved (**•)** - The IDE detects unsaved changes to your file/folder
+        - Modification (**M)** - The IDE detects a modification of existing files/folders have saved changes file or folder
+        - Untracked (**U)** - The IDE detects changes made to new files or renamed files
+- **IDE interaction**
+    - Support for Python models
+        - You can `compile` Python models to see what will ultimately get executed
+        - You can see Python models in DAG in dbt version 1.3 and higher
+    - [TESTING] Formatting: Ability to format your file powered by [sqlfmt](http://sqlfmt.com/)
+    - [TESTING] Autocomplete:
+        - [NEW] dbt autocomplete / intellisense: auto-fill model names in {{ ref ('model_name') }} or source or macro
+        - snippet autocomplete: when you type __config, we drop in a snippet of code
```

Comment:
@mirnawong1 @bossnunta Do we want to add any docs about snippets? I found this in the product feature page and wondering if we need this in the docs. I could also remove form the feature page.  https://www.notion.so/dbtlabs/IDE-Snippets-Shortcuts-f7706c97a1bc418e8170399b94cc31b5

Reply:
I don't think we need to to be honest... we'll keep updating this list, and it might be difficult to maintain.

Reply:
Let me know if you would like to push back though